nexus/src/app/sagas/mod.rs

```diff
@@ -9,8 +9,19 @@
 // correctness, idempotence, etc.  The more constrained this interface is, the
 // easier it will be to test, version, and update in deployed systems.
 
+use crate::db;
```

Comment:
I gather the new functions in this file moved here because they're now used by both the disk-create saga and the snapshot-create saga, so it doesn't make sense for them to be in either saga's module itself.  I wonder if there's a more sustainable way to share code between sagas.  Right now this file only has things that I think are potentially applicable to _all_ sagas and running sagas in general.  Maybe a "common-storage.rs" in this directory?  I don't know.

Reply:
Yeah, I agree - done in a554ef53